packages/hash/frontend/src/blocks/page/createEditorView.ts

```diff
@@ -137,7 +137,7 @@ export const createEditorView = (
 
   /** note that {@link ProsemirrorManager#defineBlock} is idempotent */
   manager.defineBlock(paragraphBlock);
-  blocksArray.forEach((block) => manager.defineBlock(block));
+  for (const block of blocksArray) manager.defineBlock(block);
```

Comment:
Minor / unrelated to this PR: WDYT of enabling the [`curly`](https://eslint.org/docs/latest/rules/curly) rule? It would auto-format this line as:

```suggestion
  for (const block of blocksArray) {
    manager.defineBlock(block);
  }
```

(same for one-line `if` statements etc.)

The result looks more verbose, but the _rhythm_ of the code becomes more consistent. I find it easier to find early `return` statements and other cases with the rule, so keen to enable it in the codebase if it is considered useful by others.